go-controller/pkg/testing/mock_ovn.go

```diff
@@ -78,9 +80,22 @@ func NewMockOVNClient(db string) *MockOVNClient {
 	mock.cache[ChassisType] = make(MockObjectCacheByName)
 	mock.cache[LogicalSwitchType] = make(MockObjectCacheByName)
 	mock.cache[ChassisPrivateType] = make(MockObjectCacheByName)
+	mock.cache[ACLType] = make(MockObjectCacheByName)
+	mock.cache[PortGroupType] = make(MockObjectCacheByName)
 	return mock
 }
 
+// Debug function used to dump the mock cache
+func PrintCache(client *MockOVNClient) {
```

Comment:
is this just for debugging? I don't see it used anywhere else. Do we actually have the capability to assert the mock cache contains what we expect it to when we run unit tests? Or are we limited right now to just checking if number of call is correct?

Reply:
This is just for debugging.  I found it useful to debug my go-ovn mock code, and I figured others could use it too.  I can remove it if you prefer.  You raise a good point, though --  we could probably improve the integration test cases by asserting that the mock cache contains what we expect.  However, the test code does not currently do that.

Reply:
should we do that as a follow up? I think it would be great if we could build some framework around that which other folks could leverage as part of dropping the current fexec expected nbctl cmds. basically check does mock cache == desired end state

Reply:
Agreed we definitely need better testing for go-ovn code moving forward.  Per offline discussion, there are multiple ways to do it.